src/core/definitions.jl

```diff
@@ -37,7 +37,8 @@ const JuMPFloatMatrix = DenseAxisArray{Float64, 2}
 const JuMPFloatArray = DenseAxisArray{Float64}
 const JuMPVariableArray = DenseAxisArray{JuMP.VariableRef}
 
-const TwoTerminalHVDCTypes = Union{PSY.TwoTerminalHVDCLine, PSY.TwoTerminalVSCDCLine}
+const TwoTerminalHVDCTypes =
```

Comment:
we can eliminate this and now use an abstract type in PSY 5

Reply:
I just merged these definitions for PSY5. I am ok merging this and making the changes in a follow up PR